.github/workflows/labeler.yml

```
@@ -0,0 +1,17 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target:
+    types: [opened, edited, synchronize]
+
+jobs:
+  labeler:
+    permissions:
+      contents: read
+      issue: write
```

actionlint (1.7.4)

unknown permission scope "issue". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)